scripts/setup_dynamodb_locally.rb

```diff
@@ -0,0 +1,65 @@
+begin
+  Exercism.config.dynamodb_client.delete_table(
+    table_name: "boffin_jobs",
+  )
+rescue Aws::DynamoDB::Errors::ResourceNotFoundException
+end
+
+Exercism.config.dynamodb_client.create_table(
+  table_name: "boffin_jobs",
+  attribute_definitions: [
+    {
+      attribute_name: "id", 
+      attribute_type: "S", 
+    },
+
+  ],
+  key_schema: [
+    {
+      attribute_name: "id", 
+      key_type: "HASH", 
+    }
+  ],
+  provisioned_throughput: {
+    read_capacity_units: 1, 
+    write_capacity_units: 1, 
+  }, 
+)
+
+Exercism.config.dynamodb_client.update_table(
```

Comment:
Interesting. You both create _and_ update the table. Is that because the table creation cannot take all the parameters the `update_table` can?

Reply:
@ErikSchierboom Yep! Very weird isn't it! 

The "attributes" can only be for the main index or for the secondary, so to add a secondary that doesn't have the same attributes as the main, you have to do two things. I might, of course, be missing something, but this is the only way I could get it working!